results/server/main.py

```diff
@@ -0,0 +1,38 @@
+import grpc
+from concurrent import futures
+from result_accumulator import ResultAccumulator
+from generated import results_pb2_grpc
+from tag_tunnel import TagTunnel
+
+tags_fp="./results/server/tags.yaml"
+tags_schema_fp="./results/server/schema/tags_schema.json"
+template_fp="./results/server/tm.py.j2"
+historic_tests_fp="../macfe-hil.github.io/historic_tests.json"
+reports_dir="../macfe-hil.github.io/reports"
+pages_repo_dir="../macfe-hil.github.io"
+pages_branch="main"
+git_username="macformularacing"
+git_email="macformulaelectric@gmail.com"
+
+def serve():
+    port = "8080"
```

Comment:
Lets change this to something that will not clash as often 
```python
port = "31763"
```

Reply:
I like the idea of adding this and all the other config parameters to the same config.yml as what is used in the hil app